test_unstructured/partition/test_xlsx.py

```diff
@@ -321,15 +321,6 @@ def test_partition_xlsx_with_more_than_1k_cells():
 class Describe_XlsxPartitionerOptions:
     """Unit-test suite for `unstructured.partition.xlsx._XlsxPartitionerOptions` objects."""
 
-    @pytest.mark.parametrize("arg_value", [True, False])
-    def it_knows_whether_to_detect_language_for_each_element_individually(
```

Comment:
Isn't this test still relevant?

Reply:
It is for `partition_xlsx()` and is still there for that case.

But `XlsxPartitionerOptions` no longer takes a `languages` arg and doesn't provide access to that value or any values derived from it, so this unit test goes away.